packages/pyright-internal/src/common/uri/emptyUri.ts

```diff
@@ -104,10 +100,6 @@ export class EmptyUri extends BaseUri {
         return 0;
     }
 
-    override combinePaths(...paths: string[]): Uri {
```

Comment:
Essentially combinePaths with an empty string has to create a file URI

Reply:
If you look at the logic in combinePaths for fileUris, it's the same as what we used to do in 'resolvePaths'